CMakeLists.txt

```diff
@@ -1161,6 +1165,10 @@ if(INSTALL_TESTS AND OPENCV_TEST_DATA_PATH)
   endif()
 endif()
 
+if(${CMAKE_SYSTEM_NAME} MATCHES QNX)
+  install(DIRECTORY ${CMAKE_BINARY_DIR}/bin/ DESTINATION bin/opencv_tests)
```

Comment:
It doesn't look correct.
Why do we need that?

Reply:
Hello, thanks for taking time to review my PR.

For QNX, we are cross-compiling, so the test binaries cannot be run on the same machine they were built on. We mostly use Ubuntu to build the binaries. So we need to install the test binaries to our SDP (Software Development Platform which is like an SDK with headers and libraries for QNX) so that they can be conveniently moved over to a QNX target.

Reply:
https://github.com/opencv/opencv/blob/4.10.0/CMakeLists.txt#L534

Reply:
Problem is still here.

Reply:
Thanks! I removed the QNX installation, and it's now using `INSTALL_TESTS` to do that.